notebooks/02_model/mmlspark_lightgbm_criteo.ipynb

```diff
@@ -0,0 +1,344 @@
+{
```

Comment:
<p>What is dbutils?</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/680/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
dbutils is for databricks, I set dbutils to None when we are on the dsvm